### ISSUE TITLE
Diagrams that produced bythis lib is not accessible for screen reader users (blind and visually impaired)

### ISSUE THREAD

**Issue by hamidrezaabroshan** [bpmn-io/diagram-js#634]
I'm a blind java developer and using Nvda screen reader to work with computer.
The diagrams that produced by this module is not accessible for screen readers E.G, by navigating the diagram, Screen reader only reads the diagram like a textual component or reads nothing at all.
Its because accessibility information not provided.
As this module is used for displaying bpmn files in bpmn modoler apps such as camunda modeler,  it effects on my job as a developer.
You can provide accessibility info in canvas by using ARIA  tag and alt text.
see <a href="https://pauljadam.com/demos/canvas.html"> accessibility in canvas </a> and
<a href="https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/"> accessibility in svg </a>
and you can download free and open source screen reader <a href="https://www.nvaccess.org/"> nvda </a> for testing accessibility.
The keywords are accessibility and screen readers.
I'm up for any help as much as I can.

1. download and run Nvda screen reader.
2. open a diagram.
3. try to understand the diagram by navigating with arrow keys and tab.

__Expected Behavior__
1. First, the diagram should be navigable by arrow keys or some shortcuts.
2. the diagram details should be provided as accessibility info.
__Environment__

 - Browser: firefox 99.
 - OS: windows 10
 - Library version: unknown.

**Comment by pinussilvestrus** [comment]
Hello @hamidrezaabroshan, thank you very much for opening this issue and providing some details! That's pretty helpful to understand the full use case.

We already have an open issue to track this problem: https://github.com/bpmn-io/diagram-js/issues/567. I think the requirements in that issue are already covering the ones that you are mentioning. Therefore, I would close this issue here as a duplicate to focus the discussions and investigations in a single place. 

If there is anything else that needs to be considered, feel free to add 👍

**Comment by hamidrezaabroshan** [comment]
Hi @pinussilvestrus, 
That's ok.
Thanks.
